pkg/controllers/dynakube/oneagent/daemonset/daemonset.go

```diff
@@ -42,6 +44,8 @@ const (
 
 	probeMaxInitialDelay         = int32(90)
 	probeDefaultSuccessThreshold = int32(1)
+
+	readOnlyRootFsVersionThreshold = "v1.291"
```

Comment:
it's not a threshold, it's aboslutly valid sem version.
wdyt to rename it to: `readOnlyRootFsMinVersion`?

Reply:
>Range constraint
> Constraint will be satisfied by versions matching >=1.291.0 <1.292.0-0.
> 1.291 is a range constraint. It means that it will match several versions.

https://jubianchi.github.io/semver-check/#/constraint/1.291

Reply:
I agree with the name `readOnlyRootFsConstraint` nice catch :+1:

Reply:
done